docuilib/src/utils/componentUtils.ts

```diff
@@ -3,5 +3,6 @@ import ReactLiveScope from '../theme/ReactLiveScope';
 const supportedComponentNames = Object.keys(ReactLiveScope);
 
 export const isComponentSupported = componentName => {
-  return supportedComponentNames.includes(componentName);
```

Comment:
I now see we have this check in several places and each time we use something different 
When I fixed the issue for Incubator components [here](https://github.com/wix/react-native-ui-lib/blob/master/docuilib/src/components/pageComponents/ContentItem.tsx#L119) I used lodash `_.get` for that

Can we unify the solution and use the util where I used `_.get` directly